db/migrate/20181109025118_add_party_ref_to_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddPartyRefToSubmissions < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :submissions, :party, foreign_key: true
```

Comment:
This should probably have a `NOT NULL` constraint on it too since we don't want rough submissions without a party 👀